packages/shadydom/src/patch-events.js

```diff
@@ -43,22 +43,23 @@ const supportsEventOptions = (() => {
 })();
 
 const parseEventOptions = (optionsOrCapture) => {
-  let capture, once, passive, shadyTarget;
-  if (optionsOrCapture && typeof optionsOrCapture === 'object') {
+  let capture, once, shadyTarget;
+  if (
+    optionsOrCapture &&
```

Comment:
super nit: this only value that `optionsOrCapture &&` is guarding against now is `null`. You could do an explicit check:
```suggestion
    optionsOrCapture !== null &&
```